src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Locale;
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+
+        return String.valueOf(Color.values()[new Random().nextInt(Color.values().length)])
```

Comment:
```suggestion
        return String.valueOf(Color.values()[new Random().nextInt(Color.values().length)])
```

lets make Random as class level variable, not to create new instance of Random class every time we call getRandomColor() method